app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -47,11 +46,11 @@ import ch.hikemate.app.ui.navigation.NavigationActions
 import ch.hikemate.app.ui.navigation.Route
 import ch.hikemate.app.ui.navigation.Screen
 import ch.hikemate.app.ui.navigation.SideBarNavigation
+import ch.hikemate.app.utils.MapUtils.showHikeOnMap
```

Comment:
Change: don't import the function directly, import `MapUtils` and use `MapUtils.showHikeOnMap` instead.

Reply:
Will do, but could you just explain to me whats the benefit? 😅

Reply:
done

Reply:
done

Reply:
I believe importing `MapUtils` and using `MapUtils.func()` improves readability, because you immediately know this is a util function that comes from another file, you don't look around in that file for this function, and you expect it to be generic to be used somewhere else.